<?xml version="1.0" encoding="utf-8"?>
<launch>
    <node pkg="sawyer_control" type="actions_subscriber.py" name="actions"/>
    <node pkg="sawyer_control" type="observations_server.py" name="observation" />
    <node pkg="sawyer_control" type="jacobian_server.py" name="jacobians" />
    <node pkg="sawyer_control" type="ik_server.py" name="inverse_kinematics" />
    <node pkg="sawyer_control" type="angle_action_subscriber.py" name="angle_action" />
</launch>
